scripts/prepare_sector_network.py

```diff
@@ -1324,6 +1324,24 @@ def add_storage_and_grids(n, costs):
             n, "H2 pipeline ", carriers=["DC", "gas pipeline"]
         )
 
+        h2_pipes["p_nom"] = 0.0
+
+        if snakemake.input.get("custom_h2_pipelines"):
+            fn = snakemake.input.custom_h2_pipelines
+            wkn = pd.read_csv(fn, index_col=0)
+
+            h2_pipes = pd.concat([h2_pipes, wkn])
```

Comment:
```suggestion
            custom_pipes = pd.read_csv(fn, index_col=0)

            h2_pipes = pd.concat([h2_pipes, custom_pipes])
```
rename to make it more general